src/main/java/LevelController.java

```diff
@@ -0,0 +1,88 @@
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.canvas.Canvas;
+import javafx.scene.layout.StackPane;
+
+import java.io.File;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.ResourceBundle;
+
+/**
+ * Created by Jim on 9/5/2015.
+ */
+public class LevelController implements Initializable {
+
+    /**
+     * The StackPane the level is drawn on.
```

Comment:
Already made an issue for changing the configuration, but I think that we should also comment private methods because others (like our TA) will be looking at our code. Making JavaDoc comments might be useful then.